azure-pipelines/pull-request-validation/pr-msal.yml

```diff
@@ -62,7 +62,13 @@ jobs:
   - task: Gradle@2
     displayName: Run Unit tests
     inputs:
-      tasks: msal:testLocalDebugUnitTest -Plabtest -ProbolectricSdkVersion=${{variables.robolectricSdkVersion}}
+      tasks: msal:testLocalDebugUnitTest -Plabtest -ProbolectricSdkVersion=${{variables.robolectricSdkVersion}} --tests *com.microsoft.identity.client*
```

Comment:
@shahzaibj and @fadidurah can you review this? As this is a change to your test configuration, CI, etc.

Reply:
I haven't seen --tests used before, but i'm guessing you're just spliting the test suite into native auth and non-native auth, which seems fine to me

Reply:
yes, this runs tests in the specified package.